deephyper/search/nas/model/space/op/gnn.py

```diff
@@ -0,0 +1,507 @@
+import tensorflow as tf
+from numpy.random import seed
+from tensorflow.keras import activations, initializers, regularizers, constraints
+from tensorflow.keras.layers import LeakyReLU, Dropout, Dense
+import tensorflow.keras.backend as K
+
+seed(2020)

```

Comment:
Setting the global random seed is dangerous because it can modify the one set somewhere else or be re-affected afterward, it is better to use RandomState: https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.random.RandomState.html.